src/types/index.ts

```diff
@@ -0,0 +1,36 @@
+export interface CategoryTheme {
+  themeId: number;
+  name: string;
+  image: string;
+}
+
+export interface PriceInfo {
+  basicPrice: number;
+  discountRate: number;
+  sellingPrice: number;
+}
+
+export interface BrandInfo {
+  id: number;
+  name: string;
+  imageURL: string;
+}
+
+export interface Product {
+  id: number;
+  name: string;
+  imageURL: string;
+  price: PriceInfo;
+  brandInfo: BrandInfo;
+}
+
+export interface Tab {
+  id: string;
+  label: string;
+  icon?: string;
+}
```

Comment:
Tab 목록이 동적이지 않고, 4가지 타입으로 고정된 경우라면 좀더 안전한 방법이 있을 것 같아요.
```suggestion
export type TabId = 'all' | 'female' | 'male' | 'teen';

export interface Tab {
  id: TabId;
  label: string;
  icon?: string;
}
```
이렇게 tabid의 4가지 타입을 각각 지정하는 것도 가능하구요,

```
export const tabs = [
  { id: 'all', label: '전체', icon: 'ALL' },
  { id: 'female', label: '여성이', icon: '👩' },
  { id: 'male', label: '남성이', icon: '👨' },
  { id: 'teen', label: '청소년이', icon: '🧑' },
] as const;
// mockData.ts

export type Tab = typeof tabs[number];
export type TabId = Tab['id'];
```
이처럼 as const 타입 단언을 이용하면 보다 안전하고 간결하게 사용할 수 있을 것 같아요!
아래의 Filter도 마찬가지겠네요.